### ISSUE TITLE
Build failed with Xcode 12.5

### ISSUE THREAD

**Issue by vitali-kastsiuk** [wuxudong/react-native-charts-wrapper#801]
Undefined symbols for architecture x86_64:
 “Swift._ArrayBuffer._copyContents(initializing: Swift.UnsafeMutableBufferPointer<A>) -> (Swift.IndexingIterator<Swift._ArrayBuffer<A>>, Swift.Int)“, referenced from:
   generic specialization <serialized, Swift._ArrayBuffer<Swift.Int8>> of Swift._copyCollectionToContiguousArray<A where A: Swift.Collection>(A) -> Swift.ContiguousArray<A.Element> in libCharts.a(ChartUtils.o)

**Comment by mateogianolio** [comment]
I have the exact same error with react-native 0.64.0. Tried with both Charts 3.3.0 and 3.6.0 and both 0.5.7 and master branch without success. Does not matter if the target is `x86_64` or `arm64`.

**Comment by smadan** [comment]
Yep, just spent 2-3 hours narrowing down the issue to this library. Removing this library allowed my app to run. Almost certainly an issue with the underlying Charts library.

**Comment by smadan** [comment]
I reported this issue there as well: https://github.com/danielgindi/Charts/issues/4636.

**Comment by Zestpond** [comment]
same issue

**Comment by mateogianolio** [comment]
Found a solution: https://github.com/Iterable/react-native-sdk/issues/86#issuecomment-744829518

**Comment by my-name-is-nheo** [comment]
@mateogianolio I'm surprised the solution is not working for everyone. reproducing the steps just caused more errors on my end

For those who are running to this issue, what is your short term solution? This library plays a major role in my app, and I'm a bit worried seeing the last update was several months ago.

**Comment by mateogianolio** [comment]
Yeah, unfortunately I have no idea why it works for me and not for others.

**Comment by vitali-kastsiuk** [comment]
@mateogianolio @my-name-is-nheo it works for me too

**Comment by smadan** [comment]
I figured out why. I needed to set ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES to YES. Details here: https://github.com/facebook/react-native/issues/31179#issuecomment-829536845.

**Comment by iranaahsanali** [comment]
I am facing the same issue on Xcode 12.5

**Comment by varunwyest** [comment]
is there a solution, we are using Xcode 12.5 and we have react native 64.1 everything else is working great. Charts needs to be upgraded. Has anybody done the upgrade in their fork?

**Comment by viet97** [comment]
any solution ???

**Comment by viet97** [comment]
> is there a solution, we are using Xcode 12.5 and we have react native 64.1 everything else is working great. Charts needs to be upgraded. Has anybody done the upgrade in their fork?

https://github.com/facebook/react-native/issues/31179#issuecomment-829536845
Check the third that is the solution
